agot-bg-game-server/src/server/serializedGameMigrations.ts

```diff
@@ -237,6 +237,27 @@ const serializedGameMigrations: {version: string; migrate: (serializeGamed: any)
                 }
             }
 
+            return serializedGame;
+        }
+    },
+    {
+        version: "7",
+        migrate: (serializedGame: any) => {
+            // Migration for #590
+            if (serializedGame.childGameState.type == "ingame") {
```

Comment:
Can you review this specific migration? It should migrate games which are in `ChooseRavenActionGameState` to `ReplaceOrderGameState`.

Reply:
Ok for me!